python/cudf/cudf/pandas/scripts/run-pandas-tests.sh

```diff
@@ -22,19 +22,6 @@ set -euo pipefail
 # of Pandas installed.
 PANDAS_VERSION=$(python -c "import pandas; print(pandas.__version__)")
 
-PYTEST_IGNORES="--ignore=tests/io/test_user_agent.py \
```

Comment:
I for sure know we have hangs in this group of tests, we will need to investigate those hangs hence opened separate issues for each one so that it's easier to enable and check each file/directory separately

Reply:
Yeah although locally I am not seeing the hangs, I can see it in this PR now. I _may_ have it narrowed it down to `tests/tslibs/test_parsing.py`